hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -46,6 +47,8 @@
   public static final String INLINE_COMPACT_PROP = "hoodie.compact.inline";
   // Run a compaction every N delta commits
   public static final String INLINE_COMPACT_NUM_DELTA_COMMITS_PROP = "hoodie.compact.inline.max.delta.commits";
+  public static final String INLINE_COMPACT_ELAPSED_TIME_PROP = "hoodie.compact.inline.max.delta.time";
```

Comment:
how about renaming it to `hoodie.compact.inline.max.delta.seconds`, it seems more readable cc @yanghua

Reply:
sounds good~

Reply:
edited. 
anything else  need to improve?

Reply:
thanks for your patient, will do a final check tomorrow.